Makefile

```diff
@@ -31,13 +31,27 @@ Makefile.inc: configure
 	@echo "$@ generated. Run the make again."
 	@exit 1
 
-install: Makefile.inc
+hwdata.pc: hwdata.pc.in
+	datadir="$(datadir)"; \
+	if [ "$${datadir#$(prefix)}" != "$$datadir" ]; then \
+		datadir="\$${prefix}$${datadir#$(prefix)}"; \
+	fi; \
+	sed -e 's|@prefix@|$(prefix)|g' \
+		-e "s|@datadir@|$$datadir|g" \
+		-e 's|@pkgdatadir@|$${datadir}/$(NAME)|g' \
+		-e 's|@VERSION@|$(VERSION)|g' \
+		-e 's|@NAME@|$(NAME)|g' \
+		hwdata.pc.in > hwdata.pc
```

Comment:
you can use `$<` instead of `hwdata.pc.in`, and `$@` instead of `hwdata.pc`; the result is the same, without hardcoding "hwdata.pc*" more